app/helpers/answer_helper.rb

```diff
@@ -6,7 +6,8 @@ def format_answer(answer)
     when Date
       answer = answer.to_formatted_s(:govuk_date)
     when Time
-      answer = answer.to_formatted_s(:govuk_time)
+      time_zone = @wizard.export_data["time_zone"] || "London" # rubocop:disable Rails/HelperInstanceVariable
```

Comment:
`@wizard.export_data` is potentially expensive - its probably better to expose a `timezone` method on the wizard - which in turn uses fetch to retrieve a key from the store.

Its probably not ideal calling a controller variable it from the view helper - in theory they should be unaware of the context outside of their parameters, to allow easy reuse.

I18n.timezone is thread local - it would be better to set it there in a `before_action` in the controller (and back to its original value in an `after_action`

That way all the time zone aware helpers would just 'behave' ?

Reply:
That's a nice suggestion, I'll look into configuring the timezone for the context of the request 👍